source/frontend/src/features/management/activities/list/ManagementActivitiesListView.tsx

```diff
@@ -83,6 +92,21 @@ export const ManagementActivitiesListView: React.FC<unknown> = () => {
     }
   }, [dataOverviewReport, overviewStatus, setDisplayModal, setModalContent]);
 
+  useEffect(() => {
+    if (invoiceStatus === 204) {
+      setModalContent({
+        variant: 'error',
+        title: 'Warning',
+        message: 'There is no data for the input parameters you entered.',
+        okButtonText: 'Close',
+        handleOk: () => setDisplayModal(false),
+      });
+      setDisplayModal(true);
```

Comment:
in my test of this logic, this modal was not able to be dismissed. Also, the warning variant may be more appropriate here. Let me know if you need reproduction steps.

Reply:
hmm - I'll look into it. I copied this from another export logic in another list view